wct-core/src/test/java/org/webcurator/core/profiles/Heritrix3ProfileTest.java

```diff
@@ -408,4 +414,94 @@ public final void testToProfileXmlHeritrix3Profile() {
         assertEquals(900L, modifiedPolitenessOptions.getRespectCrawlDelayUpToSeconds());
         assertEquals(400L, modifiedPolitenessOptions.getMaxPerHostBandwidthUsageKbSec());
     }
+
+    @Test
+    public final void testAllProfileOverrides() {
+        ProfileOverrides profileOverrides = new ProfileOverrides();
+        Heritrix3Profile profile = new Heritrix3Profile();
+        // Set the profileOverrides data
+        // No overrides set to true
+        long modifiedDocumentLimit = 25;
+        double modifiedDataLimit = 100.0d;
+        double modifiedTimeLimit = 250.0d;
+        long modifiedMaxPathDepth = 150;
+        long modifiedMaxHops = 50;
+        long modifiedMaxTransitiveHops = 5;
+        String modifiedIgnoreRobots = "ignore";
+        boolean modifiedIgnoreCookies = true;
+        List<String> modifiedBlockUrls = new ArrayList<String>();
```

Comment:
Barely worth mentioning, but the ProfileOverrides class implements the URL lists as Linked Lists, and here they're Array Lists.